ferryjs/ferry
# Naming conventions and tests

#### What's this PR do?

Per #6, this path renames the core classes accordingly:

Adapter -> Specifiction
Engine -> Router
ORM -> Storage

This patch also introduces coverage for the Specification & Storage classes and partial coverage for the Ferry class.
#### How should this be manually tested?
- `npm install`
- `npm test` confirm that all tests run and pass
- `cd examples/simple && npm start`
- open new terminal window and run `curl http://localhost:3333/api/v1/pets`.  This should return 200 and and "[]"
#### Any background context you want to provide?

See #6 
#### What are the relevant tickets?
#6
#### Screenshots (if appropriate)
#### Questions:
#### Checklist:
- [ ] Code standard review
- [ ] End user documentation provided
#### Release Type:
